python/paddle/fluid/tests/unittests/parallel_executor_test_base.py

```diff
@@ -41,6 +41,7 @@ def check_network_convergence(self,
                                   use_parallel_executor=True,
                                   use_reduce=False,
                                   use_ir_memory_optimize=False,
+                                  enable_inplace=False,
```

Comment:
True?

Reply:
test on offline models. So I didn't turn it on. Should we turn it ?

Reply:
yes

Reply:
done.